codegen.yml

```diff
@@ -0,0 +1,23 @@
+overwrite: true
+schema: "http://localhost:8080/graphql"
+documents: './src/app/graphql/queries.ts'
+generates:
+  ./src/app/graphql/types.tsx:
+    plugins:
+      - add:
+          content:
+          - ''
+          - 'export namespace GraphQL {'
```

Comment:
We really don't need extra namespace `GraphQL` . All hooks and types are exported itself.